### PR TITLE
multiple files + labels extraction

### PR DESCRIPTION
**Pull request by p0l0satik**
formated cpf

### CODE REVIEW

File: cpf_segmentation/Dockerfile [hunk 1]
```diff
@@ -2,8 +2,11 @@ FROM ubuntu:18.04
 
 ARG DEBIAN_FRONTEND=noninteractive
 
-RUN apt-get update
-RUN apt-get install -y cmake pkg-config libeigen3-dev libopencv-dev libpcl-dev
+RUN apt-get update && apt-get install -y python3-pip git
+
+RUN python3 -m pip install --user git+https://github.com/DanielPollithy/pypcd.git
```
Comment (DmiitriyJarosh) [inline cpf_segmentation/Dockerfile:7]: simple `pip install pypcd` doesn't work?

File: cpf_segmentation/src/segmentation_test.cpp [hunk 2]
```diff
@@ -100,22 +100,18 @@ int main(int argc, char** argv){
...
 
-    size_t dot = outputname.find_last_of ('.');
-    if (dot != std::string::npos)
+  std::string outputname = argv[1];
+
+  size_t dot = outputname.find_last_of ('.');
+  if (dot != std::string::npos)
     outputname = outputname.substr (0, dot);
+  outputname = "output/" + outputname + "/" + outputname;
+  outputname+="_seg.pcd";
```
Comment (DmiitriyJarosh) [inline cpf_segmentation/src/segmentation_test.cpp]: Let's remove this suffix

File: cpf_segmentation/Dockerfile (hunk 1 above)
Comment (p0l0satik) [reply cpf_segmentation/Dockerfile:7]: unfortunately no, it requires this specific installation to get the right version

File: cpf_segmentation/src/segmentation_test.cpp (hunk 2 above)
Comment (p0l0satik) [reply cpf_segmentation/src/segmentation_test.cpp]: done

**Review by DmiitriyJarosh — APPROVED** [review]

**Pull request merged**